configmap-reload.go

```diff
@@ -191,29 +188,53 @@ func renderConfigs(outputVolumeDir, volumeDir, secretPath string) error {
 		}
 	}
 
-	err = filepath.Walk(volumeDirs[0], func(path string, info os.FileInfo, err error) error {
+	err = filepath.Walk(volumeDir, func(path string, info os.FileInfo, err error) error {
 		// failed to access the file, or some other unworkaroundable error
 		if err != nil {
-			return err
+			return fmt.Errorf("walk err: %s", err)
+		}
+
+		// support symbolic links
+		if info.Mode()&os.ModeSymlink != 0 {
```

Comment:
I've never seen this syntax before! It looks pretty fancy, but I think it's less intuitive than just using a standard comparison here:
```go
if info.Mode() == os.ModeSymlink {
    ...
```